Error: Cannot find module 'broccoli-global-exporter'

It seems like version `0.6.3` is missing the `dist/` folder.

Considering this:

```
cd
mkdir test-broccoli-global-exporter && cd $_
npm install broccoli-global-exporter
```

Installation worked fine:

```
broccoli-global-exporter@0.6.3 node_modules/broccoli-global-exporter
├── lodash.first@3.0.0
├── lodash.defaults@3.1.2 (lodash.restparam@3.6.1)
└── lodash.isplainobject@3.2.0 (lodash._basefor@3.0.3, lodash.isarguments@3.0.8, lodash.keysin@3.0.8)
```

index.js:

``` js
console.log(require('broccoli-global-exporter'));
```

`node index.js` produces an error:

```
module.js:338
    throw err;
          ^
Error: Cannot find module 'broccoli-global-exporter'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (~/test-broccoli-global-exporter/index.js:1:75)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

It seems that the `dist/` folder is missing from the NPM package:

`find node_modules/broccoli-global-exporter/dist`:

```
find: node_modules/broccoli-global-exporter/dist: No such file or directory
```

My guess is that `npm prepublish` creates the dist folder correctly, but `npm pack` ignores it somehow.

--- a/.npmignore
+++ b/.npmignore
@@ -1,0 +1,3 @@
+/node_modules
+/coverage
+/specs

